### DISCUSSION TITLE
Custom plug-in path provided in codecept.conf.js gives error as Could not load plugin

### DISCUSSION THREAD

**Discussion by raghavendra-polishetty** [codeceptjs/CodeceptJS#3279] [Q&A]
I tried to provide the custom plug-in path in codecept.conf.js as : 

    plug-in-name: {
     require: './node_modules/<plug-in-name>.ts',
      enabled : true
    }

gives error as Could not load plugin. 
How to specify exact path location of custom plug-in in codecept.conf.js file ?

**Comment by kobenguyent** [comment]
I'm curious why the plugin is inside node_modules?
